<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDlgSettingsLibVnc</class>
 <widget class="QDialog" name="CDlgSettingsLibVnc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setting LibVnc</string>
  </property>
  <property name="windowIcon">
   <iconset theme="app"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbOk">
       <property name="text">
        <string>Ok(&amp;O)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel(&amp;C)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Server">
      <attribute name="title">
       <string>Server</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leName">
           <property name="toolTip">
            <string>Connecter name</string>
           </property>
           <property name="statusTip">
            <string>Connecter name</string>
           </property>
           <property name="whatsThis">
            <string>Connecter name</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="CParameterNetUI" name="wNet" native="true"/>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="cbShared">
           <property name="text">
            <string>Shared</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbOnlyView">
           <property name="text">
            <string>Only view</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="cbLocalCursor">
           <property name="text">
            <string>Use local cursor</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbClipboard">
           <property name="text">
            <string>Enable clipboard</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="cbShowServerName">
         <property name="text">
          <string>Show server name</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Compress">
      <attribute name="title">
       <string>Compress</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QCheckBox" name="cbCompress">
         <property name="text">
          <string>Compress level:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QSpinBox" name="spCompressLevel">
           <property name="maximum">
            <number>9</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>0=fast, 9=best</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QCheckBox" name="cbJPEG">
           <property name="text">
            <string>Enable JPEG compress</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spJPEGLevel">
           <property name="maximum">
            <number>9</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>0=Worst, 9=best</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>206</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CParameterNetUI</class>
   <extends>QWidget</extends>
   <header location="global">ParameterNetUI.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cbShared</tabstop>
  <tabstop>cbLocalCursor</tabstop>
  <tabstop>cbClipboard</tabstop>
  <tabstop>pbOk</tabstop>
  <tabstop>pbCancel</tabstop>
  <tabstop>cbCompress</tabstop>
  <tabstop>spCompressLevel</tabstop>
  <tabstop>cbJPEG</tabstop>
  <tabstop>spJPEGLevel</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
